app/DynamicsAdapter/DynamicsAdapter.Web/PersonSearch/PersonSearchController.cs

```diff
@@ -112,24 +122,23 @@ public async Task<IActionResult> MatchFound(Guid id, [FromBody] MatchFound match
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [Route("Event/{id}")]
-        public async Task<IActionResult> Event(Guid id, [FromBody] Object temp)
+        public async Task<IActionResult> Event(Guid id, [FromBody]ProviderSearchEventStatus providerSearchEventStatus)
         {
 
             _logger.LogInformation($"Received new event for SearchApiRequest [{id}]");
 
             var token = new CancellationTokenSource();
 
-            var searchApiEvent = new SSG_SearchApiEvent()
-            {
-                Id = Guid.NewGuid(),
-                Message = "Test Event from dynadapter",
-                ProviderName = "Dynadapter",
-                TimeStamp = DateTime.Now,
-                Type = "Test"
-            };
-
             try
             {
+                var searchApiEvent = new SSG_SearchApiEvent()
+                {
+                    Id = Guid.NewGuid(),
```

Comment:
SSG_SearchApiEvent should have connection with searchApiRequest or SearchRequest, right?

Reply:
That's pretty weird yeah, I have to put a new Id for it to work, otherwise it was creating errors.. the id is passed in the SearchApiRequestService

Reply:
`var result = await _searchApiRequestService.AddEventAsync(id, searchApiEvent, token.Token);`

Reply:
ok. saw it.